Gopkg.lock

```diff
@@ -15,7 +15,7 @@
   name = "github.com/Azure/go-ansiterm"
   packages = [
     ".",
-    "winterm",
+    "winterm"
```

Comment:
I actually downgraded to the same version as the vendor-check before doing the vendoring fixes.

Reply:
@simonferquel Did you redo the updates in the commits you cherry-picked from me? Because I have no idea which version I was running ;-). (Maybe I'll look into containerising the update...)